src/sonic_ax_impl/mibs/vendor/cisco/ciscoPfcExtMIB.py

```diff
@@ -245,7 +251,9 @@ def indications_per_priority(self, sub_id):
         if port_oid in self.oid_lag_name_map:
             counter_value = 0
             for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[port_oid]]:
-                counter_value += self._get_counter(mibs.get_index_from_str(lag_member), counter_name)
+                member_counter = self._get_counter(mibs.get_index_from_str(lag_member), counter_name)
+                if member_counter:
```

Comment:
>if [](http://example.com/codeflow?start=16&length=2)

In else branch, should we set counter_value=None, so client could still query it and understand some internal counter issue.

Reply:
I see that "if member_counter" condition will hold good if member_counter is not None or is not zero.
In case a specific counter is 0, that will also drop to else condition.
I can do either:
if member_counter:
     add ..

or 
if member_counter is not None:
    ad...
else:
   return None # this will handle the scenario where _get_counter returned None for any of its member interface.

Reply:
@SuvarnaMeenakshi , I think @qiluo-msft original comment is suggesting you to take on the second choice to return None.  Please go ahead modify your PR accordingly and we can ask @qiluo-msft to help review it again.
Thanks!

Reply:
Updated as suggested and corresponding unit-test.